llvm/test/MC/AMDGPU/gfx11_asm_vop3.s

```diff
@@ -2824,49 +2824,61 @@ v_lshrrev_b64 v[5:6], src_scc, src_scc
 v_lshrrev_b64 v[254:255], 0xaf123456, 0.5
 // GFX11: encoding: [0xfe,0x00,0x3d,0xd7,0xff,0xe0,0x01,0x00,0x56,0x34,0x12,0xaf]
 
-v_mad_i16 v5, v1, v2, s3
+v_mad_i16 v5.l, v1.l, v2.l, s3
 // GFX11: encoding: [0x05,0x00,0x53,0xd6,0x01,0x05,0x0e,0x00]
 
-v_mad_i16 v5, v255, s2, s105
-// GFX11: encoding: [0x05,0x00,0x53,0xd6,0xff,0x05,0xa4,0x01]
+v_mad_i16 v5.l, v255.h, s2, s105
+// GFX11: encoding: [0x05,0x08,0x53,0xd6,0xff,0x05,0xa4,0x01]
 
-v_mad_i16 v5, s1, v255, exec_hi
-// GFX11: encoding: [0x05,0x00,0x53,0xd6,0x01,0xfe,0xff,0x01]
+v_mad_i16 v5.l, s1, v255.h, exec_hi
+// GFX11: encoding: [0x05,0x10,0x53,0xd6,0x01,0xfe,0xff,0x01]
 
-v_mad_i16 v5, s105, s105, exec_lo
+v_mad_i16 v5.l, s105, s105, exec_lo
 // GFX11: encoding: [0x05,0x00,0x53,0xd6,0x69,0xd2,0xf8,0x01]
 
-v_mad_i16 v5, vcc_lo, ttmp15, v3
+v_mad_i16 v5.l, vcc_lo, ttmp15, v3.l
 // GFX11: encoding: [0x05,0x00,0x53,0xd6,0x6a,0xf6,0x0c,0x04]
 
-v_mad_i16 v5, vcc_hi, 0xfe0b, v255
-// GFX11: encoding: [0x05,0x00,0x53,0xd6,0x6b,0xfe,0xfd,0x07,0x0b,0xfe,0x00,0x00]
+v_mad_i16 v5.l, vcc_hi, 0xfe0b, v255.h
+// GFX11: encoding: [0x05,0x20,0x53,0xd6,0x6b,0xfe,0xfd,0x07,0x0b,0xfe,0x00,0x00]
 
-v_mad_i16 v5, ttmp15, src_scc, ttmp15
+v_mad_i16 v5.l, ttmp15, src_scc, ttmp15
 // GFX11: encoding: [0x05,0x00,0x53,0xd6,0x7b,0xfa,0xed,0x01]
 
-v_mad_i16 v5, m0, 0.5, m0
+v_mad_i16 v5.l, m0, 0.5, m0
 // GFX11: encoding: [0x05,0x00,0x53,0xd6,0x7d,0xe0,0xf5,0x01]
 
-v_mad_i16 v5, exec_lo, -1, vcc_hi
+v_mad_i16 v5.l, exec_lo, -1, vcc_hi
 // GFX11: encoding: [0x05,0x00,0x53,0xd6,0x7e,0x82,0xad,0x01]
 
-v_mad_i16 v5, exec_hi, null, vcc_lo op_sel:[1,1,1,1]
+v_mad_i16 v5.h, exec_hi, null, vcc_lo op_sel:[1,1,1,1]
 // GFX11: encoding: [0x05,0x78,0x53,0xd6,0x7f,0xf8,0xa8,0x01]
 
-v_mad_i16 v5, null, exec_lo, 0xfe0b op_sel:[0,0,0,0]
```

Comment:
I think it's ok to remove them since these are all lo16 and the semantic remain the same. The other tests with op_sel will cover the testing of the format